_posts/2020-02-17-Combining-MongoDB-and-AWS-Lambda.md

```diff
@@ -0,0 +1,310 @@
+* Account ID: your AWS account Id which you can find under the 'My Account' in the AWS console
+* VPC-id: Fill in the VPC-id that you copied from the vpc that you just created in AWS
+* VPC CIDR: specify the CIDR block that you used to configure your vpc with on AWS
+* region: the region where you created the AWS vpc
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/8-setup-peering-connection-atlas.png" width="70%" height="70%">
+</div>
+
+Hit `Instantiate peering` !
+
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/9-pending-peering-connection.png" width="70%" height="70%">
+</div>
+
+Notice that MongoDB created an `Atlas CIDR` which specifies the IP range in which your Atlas cluster will reside.  
+We are connecting the Atlas IP range with the IP range of our AWS VPC, hence VPC peering.
+
+The peering connection is now pending.   
+
+Go back to AWS.
+
+## The AWS side (again)
+In the VPC service of AWS go to `Peering Connections`.  
+You will a new peering request with stats `Pending Acceptance`.
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/10-aws-peering-connection-request.png" width="80%" height="80%">
+</div>
+
+Accept this peering request!
+
+Now you have to update your routing tables.
+AWS will also ask you `Do you want to update your routing tables` when you accept the peering request.  
+We will deploy our Lambda Functions in the public subnet of our VPC.
+So we want to modify the route table that is associated with that subnet.
+You can recognize that route table because it has an **explicit subnet association**.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/11-modify-route-table.png" width="100%" height="100%">
+</div>
+
+Click `edit routes` and add a route towards your Atlas cluster.  
+What we are actually saying here is that we want to route all traffic to our Atlas cluster through the VPC peering connection.  
+As `Destination` choose the Atlas CIDR and under `Target` choose your VPC peering connection.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/12-edit-routes.png" width="100%" height="100%">
+</div>
+
+Updating the route table will update the status of the peering connection to **available** in the Atlas UI.  
+This takes a couple of minutes.
+
+<div style="text-align: center;" >
+  <img src="/img/2020-02-17-Combining-MongoDB-Atlas-and-AWS-Lambda/13-peering-available.png" width="100%" height="100%">
+</div>
+
+### Deploy your lambda functions!
+
+Deploy your Lambda functions to your VPC and test them out!  
+To deploy them in our vpc you have to configure there VPC config:
+* use the pubic subnet that we just created
+* specify the security group of you AWS vpc
+
+With AWS `SAM` that looks like this: 
```

Comment:
Wait whut? I have no idea what to do next. What is `SAM`? Where is this yaml config coming from? How do you get to the deployed Lambda? What Lambda did you use?

Either link to other resources/blogpost/workshop material where you did this or explain how to setup a basic Lamba that tests the connection to MongoDB with the VPC peering.